news/2023/2023-08-25-osu-language-exchange-september-2023.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "osu! Language Exchange: September 2023"
+date: 2023-08-25 20:00:00 +0000
+---
+
+The best way to learn a new language is to use it, and what better way to use a language than to make some new friends?
+
+![](/wiki/shared/news/2023-08-25-osu-language-exchange-september-2023/banner.jpg)
+
+osu! Language Exchange (OLEX) is an event where you can meet people to help you practice the languages you're learning and take your language skills to the next level. The event will last from **September 1–30, 2023** and take place in the **[osu! University Discord server](https://discord.com/invite/QubdHdnBVg)**. After filling out the **[sign-up form](https://forms.gle/GiZUvqu4XQtjidMt5)**, you will be matched in groups of 2 to 4 with people who speak the language you want to learn and are currently learning a language that you speak.
+
+## Registration
+
+When signing up, you will be asked for:
+
+- **The languages you speak.** These can either be a native language or any language that you feel near-native in.
+- **1 to 2 languages you want to learn.** These are referred to as *target languages*, and you will be matched with native speakers of those languages to practice with.
+- **Your current level in your target languages.**
+- **The language skills you want to focus on most.** You can choose reading, writing, speaking, listening, or all four!
+- **Your availability throughout the month.**
+- **Your preferred learning style between text and voice.**
+
+This information will be used to help you get matched with the best group for you. If you're ready to sign up, you can fill out the sign-up form [here](https://forms.gle/GiZUvqu4XQtjidMt5).
+
+## Event information
+
+When the event begins on September 1, 2023, you'll get to meet your new language exchange partners! Based on the languages you chose in your sign-up form, you will be added to 1 to 2 groups with 2 to 4 people in each group.
+
+This is the part where you share stories, play games together, and help each other grow your language skills! There will be some suggested activities for you to do throughout the month, but it's up to you to decide how your language learning adventure will go.
+
+## Achievements
+
+Throughout the event, there will be achievements for you to complete that require you to do different activities with your language exchange partners. Some examples include having a certain amount of conversations, writing a certain amount of short stories, or even doing cooking streams in call with your language partners. The full list of achievements can be found on the event's [forum post](https://osu.ppy.sh/community/forums/topics/1808561).
```

Comment:
Cooking? Do you mean mapping?

One question I have for all of this is.. is there any osu! related anything in this event? If so it feels out of place for a news post. Like at very least the quests should be osu! related no??

Reply:
no, as in actual cooking. it would be weird to make activities that have to do with osu! in this kind of event since the focus is on learning a language, though what you mention is sort of a thing (but not everyone is a mapper)

![image](https://github.com/ppy/osu-wiki/assets/36758269/681cb830-abbc-43ba-9ca1-a16f86da4acb)

aside from being an event in the osu! community, no there's nothing specifically osu!-related here. feel free to close if that's a dealbreaker